docs/mkdocs.yml

```diff
@@ -0,0 +1,59 @@
+site_name: DJL-demo
+repo_name: aws-samples/djl-demo
+repo_url: https://github.com/aws-samples/djl-demo
+use_directory_urls: false
+markdown_extensions:
+  - codehilite
+  - toc:
+      permalink: true
+theme:
+  name: material
+  features:
+    - tabs
+  palette:
+    primary: light-blue
+    accent: light-blue
+google_analytics:
+  - UA-159559852-2
+  - auto
+docs_dir: '../'
+plugins:
+  - search
+  - mknotebooks
+  - exclude:
+      regex:
+        - '.*\.(symbol|params|pt|jar|gz|zip|txt|json|sh|bat|cmd|puml|g4|java|py|cc|h|pro|properties|xml|gradlew|gradle|Dockerfile|RepositoryFactory|EngineProvider|ZooProvider)$|(docker|integration|testing|tools)/.*$'
```

Comment:
'.*\.(*+-runner|DS_Store|MF|analysis|ap_|apk|bat|bin|cache|class|classpath|cmd|csv|dat|dat.*|dex|dumpstream|editorconfig|eslintrc|flat|gitignore|gradle|html|idx|iml|index|jar|java|jks|lock|lst|mapping|name|pack|params|pb|prefs|prettierignore|prettierrc|pro|project|properties|pt|sample|sbt|scala|ser|sh|snap|so|ts|tsx|txt|xml)$'

Reply:
Are these additions to the existing types or should I replace them ?

Reply:
those are file extensions for this repo. You need replace with new one:
```
'.*(*+-runner|DS_Store|MF|analysis|ap_|apk|bat|bin|cache|class|classpath|cmd|csv|dat|dat.*|dex|dumpstream|editorconfig|eslintrc|flat|gitignore|gradle|html|idx|iml|index|jar|java|jks|lock|lst|mapping|name|pack|params|pb|prefs|prettierignore|prettierrc|pro|project|properties|pt|sample|sbt|scala|ser|sh|snap|so|ts|tsx|txt|xml)$'
```

Reply:
So I think this is invalid. Maybe we don't need the *+ inside the file extensions group. 

Checked it here. regexr.com/597s1

Reply:
This should work:
```
.*((.+-runner)|DS_Store|MF|analysis|ap_|apk|bat|bin|cache|class|classpath|cmd|csv|dat|(dat.*)|dex|dumpstream|editorconfig|eslintrc|flat|gitignore|gradle|html|idx|iml|index|jar|java|jks|lock|lst|mapping|name|pack|params|pb|prefs|prettierignore|prettierrc|pro|project|properties|pt|sample|sbt|scala|ser|sh|snap|so|ts|tsx|txt|xml)$
```